app/ui.R

```diff
@@ -95,11 +95,15 @@ ui <- fluidPage(
             radioButtons('vaccine', "Which of the available COVID-19 vaccines did you recieve?",
                          choiceNames = unname(vaccine_labels), choiceValues = names(vaccine_labels),
                          inline=TRUE),
-            selectInput('doses', "How many doses of this vaccine have you recieved?",
-                         1:2),
+            conditionalPanel(
+              condition = "input.vaccine != 'johnsonandjohnson'",
+              selectInput('doses', "How many doses of this vaccine have you recieved?",
```

Comment:
"recieved" should be changed to "received"

Reply:
Thank you for catching!